chemprop/data/utils.py

```diff
@@ -819,7 +858,9 @@ def split_data(data: MoleculeDataset,
         test = [data[i] for i in indices[train_val_size:]]
 
         return MoleculeDataset(train), MoleculeDataset(val), MoleculeDataset(test)
-
+    elif split_type == 'molecular_weight':
+        train, val, test = weight_split(data, bias='small', sizes=sizes)
+        return train, val, test
```

Comment:
```suggestion
        return weight_split(data, bias='small', sizes=sizes)
```
the weight splitting strategy is not exposed here?